components/mpas-ocean/src/Registry.xml

```diff
@@ -984,11 +984,15 @@
 		<nml_option name="config_frazil_heat_of_fusion" type="real" default_value="3.34e5" units="J kg^-1"
 					description="Energy per kilogram released when sea water freezes. NOTE: test and make consistent with E3SM."
 					possible_values="Any positive real number."
-		/>
+		/> 
 		<nml_option name="config_frazil_ice_density" type="real" default_value="1000.0" units="kg m^-3"
 					description="Assumed density of frazil. NOTE: test and make consistent with E3SM."
 					possible_values="Any positive real number."
 		/>
+		<nml_option name="config_frazil_ice_porosity" type="real" default_value="0.85" units="1"
+					description="Assumed porosity of frazil"
+					possible_values="Any positive real number from 0 to 1."
+		/>
```

Comment:
I'm wondering if we should call this something other than porosity. When I hear porosity, I assume that the volume of frazil ice formed is only `porosity * frazil ice thickness tendency` (`* areaCell`) because part of the frazil ice tendency (the volume advected upward) is pure seawater. But my understanding of the implementation is really that this is expressing the salinity of the frazil as a fraction of the surrounding seawater (the ice formed and latent heat extracted is the full frazil ice thickness tendency). So I wonder if a less confusing name would be something like `config_frazil_ice_salinity_fraction`.

Reply:
I also got a bit confused by the use of "porosity".  I think the use of that term was partly responsible for the confusion we had in our webex discussion a little while back.  I seemingly couldn't let go of the idea that not just salt but seawater was being captured and transported by frazil in this approach.

Reply:
@njeffery, do you have feedback on this?

Reply:
@xylar  and @cbegeman : I believe the confusion comes from thinking of frazil as fresh ice + water when it's really sea ice which is a mushy layer.  The use of porosity in this context is quite common.